src/odemis/acq/stitching/_weaver.py

```diff
@@ -130,6 +133,17 @@ def get_bounding_boxes(tiles: list):
 
             tbbx_phy.append(bbx)
 
+            # add the stage-bare position
+            sbc = t.metadata.get(model.MD_STAGE_POSITION_RAW, None)
+            if sbc is not None:
+                stage_bare_coords.append(copy.deepcopy(sbc))
```

Comment:
```suggestion
                stage_bare_coords.append({**sbc})
```
Would be way more performant for this datastructure:
```
>>> meth1 = lambda lst, dct: lst.append(deepcopy(dct))
>>> meth2 = lambda lst, dct: lst.append({**dct})
>>> a = {"x": 0.0, "y": 1.0, "z": 2.0, "rx": 0.0, "rz": 0.0}
>>> dct1, dct2 = [], []
>>> timeit.timeit(functools.partial(meth1, dct1, a), number=10000)
0.2051957930016215
>>> timeit.timeit(functools.partial(meth2, dct2, a), number=10000)
0.008122990999254398
```

Reply:
It's not obvious what that unpacking function is doing, and if it is even a deep copy. prefer not to